src/main/scala/org/broadinstitute/hail/driver/FilterAlleles.scala

```diff
@@ -0,0 +1,156 @@
+
+package org.broadinstitute.hail.driver
+
+import scala.math.min
+import org.apache.spark.rdd.RDD
+import org.broadinstitute.hail.Utils._
+import org.broadinstitute.hail.annotations._
+import org.broadinstitute.hail.expr.{EvalContext, Parser, TArray, TBoolean, TGenotype, TInt, TSample, TVariant}
+import org.broadinstitute.hail.methods.Filter
+import org.broadinstitute.hail.variant.{AltAllele, GTPair, Genotype, Variant}
+import org.kohsuke.args4j.{Option => Args4jOption}
+
+import scala.collection.{GenTraversableOnce, mutable}
+import scala.reflect.ClassTag
+
+object FilterAlleles extends Command {
+  class Options extends BaseOptions {
+    @Args4jOption(required = false, name = "--keep", usage = "Keep variants matching condition")
+    var keep: Boolean = false
+
+    @Args4jOption(required = false, name = "--remove", usage = "Remove variants matching condition")
+    var remove: Boolean = false
+
+    @Args4jOption(required = true, name = "-c", aliases = Array("--condition"),
+      usage = "Filter expression involving <code>v</code> (variant), <code>va</code> (variant annotations), and <code>aIndex</code> (allele index)",
```

Comment:
Ah yeah, it probably does. I originally had back ticks but those don't get converted in the docs. I'll change it to plain text without markup.